src/HotChocolate/Stitching/src/Stitching.Redis/RedisExecutorOptionsProvider.cs

```diff
@@ -105,7 +105,8 @@ await GetRemoteSchemaDefinitionAsync(schemaName)
             IList<IConfigureRequestExecutorSetup> factoryOptions,
             CancellationToken cancellationToken)
         {
-            await using ServiceProvider services =
+            // Don't dispose of the Service Provider here. Causes ObjectDisposedExceptions in RequestExecutorResolver.BeginRunEvictionEvents!
+            ServiceProvider services =
```

Comment:
Where do we handle disposing the service provider?

Reply:
Call chain is:
https://github.com/ChilliCream/hotchocolate/blob/437e97a5764092c6a75b50b93876e010196e91db/src/HotChocolate/Core/src/Execution/RequestExecutorResolver.cs#L162

https://github.com/ChilliCream/hotchocolate/blob/437e97a5764092c6a75b50b93876e010196e91db/src/HotChocolate/Core/src/Execution/RequestExecutorResolver.cs#L461

This is currently prevented as the RegisteredExecutor.Dispose() is never called due to accessing the prematurely disposed  ServiceProvider during the Eviction Actions.